resources/telemetry/charts/fluent-bit/templates/daemonset.yaml

```diff
@@ -1,3 +1,5 @@
+{{- if not .Values.global.operatorManagedFluentBit }}
+{{- if .Values.networkPolicy.enabled }}
```

Comment:
I don't think it should depend on the `networkPolicy.enabled`

Reply:
Don't know how this sneaked in 😆 Removed